README.md

```diff
@@ -13,7 +13,7 @@ This repo contains libraries that enable a **light-weight scripting model** for
 
 These libraries are the runtime used by [Azure Functions](https://azure.microsoft.com/en-us/services/functions/). The runtime builds upon the tried and true [Azure WebJobs SDK](https://github.com/Azure/azure-webjobs-sdk) - this library just layers on top to allow you to "**script the WebJobs SDK**".
 
-**An important note on language support levels** - while many languages are supported, their level of support differs in important ways, making some languages more suitable than others for certain workloads. The **first class** languages are C#, F# and Javascript/Node.js. Functions written in these languages are run **in process** and are suitable for any workload. The remaining languages are considered **experimental**. Functions written in these languages are scripts that are run **out of process**. While there are many scenarios where this is acceptable, it won't be acceptable for high load scenarios where the overhead of a new process for each invocation won't scale.
+> **An important note on language support levels** - while many languages are supported, their level of support differs in important ways, making some languages more suitable than others for certain workloads. In V1, the **first class** languages are C#, F# and Javascript/Node.js. Functions written in these languages are run **in process** and are suitable for any workload. The remaining languages are considered **experimental**. Functions written in these languages are scripts that are run **out of process**. While there are many scenarios where this is acceptable, it won't be acceptable for high load scenarios where the overhead of a new process for each invocation won't scale. In V2, running a language **out of process** is no longer considered experimental. These include JavaScript/Node.js and Java as well as languages in preview such as PowerShell and Python. C# and F# are still run **in process**. For more information about Supported Languages, see [this Microsoft Docs page](https://docs.microsoft.com/en-us/azure/azure-functions/supported-languages).
```

Comment:
A couple of notes:

- This paragraph is getting large; I would split it into a small intro paragraph, then paragraphs for V1 and V2.
- Remove the `en-us` segment from the documentation URL; this will cause the link to redirect to the native language of the reader's machine and is more friendly to our non-English-speaking users.
- "These include JavaScript/Node.js.." Wording doesn't flow as smoothly as it could; would something like "Out-of-process languages include JavaScript/Node.js (GA), Java (GA), PowerShell (preview) and Python (preview)." sound better?